order-service/src/main/java/com/learningmicroservices/orderservice/service/OrderService.java

```diff
@@ -0,0 +1,92 @@
+package com.learningmicroservices.orderservice.service;
+
+import com.learningmicroservices.orderservice.Enum;
+import com.learningmicroservices.orderservice.dto.InventoryResponse;
+import com.learningmicroservices.orderservice.dto.OrderLineItemsDto;
+import com.learningmicroservices.orderservice.dto.OrderRequest;
+import com.learningmicroservices.orderservice.event.OrderPlacedEvent;
+import com.learningmicroservices.orderservice.model.Order;
+import com.learningmicroservices.orderservice.model.OrderLineItems;
+import com.learningmicroservices.orderservice.repository.OrderRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+@Transactional
+@Slf4j
+public class OrderService {
+
+    private final OrderRepository orderRepository;
+
+    @Autowired
+    private final WebClient.Builder webClientBuilder;
```

Comment:
remove autowired annotation as it is not needed here. you are using constructor injection.